lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -903,20 +903,28 @@ static bool checkConcatNodeUniformDims(llvm::ArrayRef<NodeValue> inputs,
   return true;
 }
 
-// Given one tensor and the desired sub-tensor size, \returns the dimension,
-// after which the sub-tensor has the desired size, otherwise return -1.
-// Example: Given a tensor <2,3,4,5>, and a desired size 20, this function will
-// return dimension 1 as the sub-tensor after it is <4,5>, which has the
-// size 20.
-static ssize_t
-findMatchingConcatDimForSameSizeSubTensor(llvm::ArrayRef<size_t> firstDims,
-                                          size_t desiredSubTensorSize) {
-  size_t currentSubTensorSize = 1;
+/// Given one tensor and the desired leading/trailing dimensions sizes.
```

Comment:
Can you specify the arguments here in the description for doxygen? E.g. `Given a tensor's dims \p firstDims, ...` etc.

Reply:
thx. done